### ISSUE TITLE
Feature: KBIN API Integation // Merged view of Communities & Magazines

### ISSUE THREAD

**Issue by samuk** [tgxn/lemmy-explorer#119]
**What do you want to see?**
A clear and concise description of what you would like to see implemented.

I would like to see both communities and magazines displayed in a single view.

**Additional context**

I can subscribe to either, I don't really care if it's a magazine or a community.

**Comment by tgxn** [comment]
The main reasons I separated these are because the API isn't stable, and the data that is available for each is vastly different _(ie. for communities I can get growth data, user activity, post counts, etc; for kbin, there is nothing)_

I don't have a way to integrate these two lists without a way to sort kbin instances into them (having data on user/description/banner etc)

**Comment by tgxn** [comment]
Also, the kbin crawling is using [this script](https://github.com/tgxn/lemmy-explorer/issues/100#issuecomment-1617444934) which is not ideal. without these features improving in kbin, it's a bit hard to manage.

**Comment by samuk** [comment]
Hopefully this WIP API pull will make that possible https://codeberg.org/Kbin/kbin-core/pulls/357

**Comment by tgxn** [comment]
#150 relates to missing some kbin instances, likely due to the poor crawling method.

when the api is updated and the instances are merged, we will be able to have the same or similar filters as the ### community list.

**Comment by tgxn** [comment]
does this software report software: `kbin` or `mbin` ? I'd have to add another scraper for these.. :/

**Comment by tgxn** [comment]
@FitikWasTaken have been busy with some other projects lately, and any kbin/mbin integration has taken a backseat, since the primary goal of this project is still with Lemmy.

I'd have to investigate their api and write a custom handler for something that's currently only deployed on a small percentage of the fediverse... within the next year :D

**Comment by tgxn** [comment]
Looks like the Magazine crawling has once again broken :(

Will have to look and see how the API changes are going for KBIN.

**Comment by tgxn** [comment]
Yeah it seems that it's completely dead and instances have mostly stopped returning results. :/

**Comment by tgxn** [comment]
okay, i can see there's 17 mbin and 1 kbin instance in my data (still active in the last 2 days)

at least they have an API!

there are only ~410 magazines across ~31 instances.

![image](https://github.com/user-attachments/assets/a6f3a825-e5f7-40a6-8ce7-0b49d8142ef8)
